### PR TITLE
pyobjc-core 8.5 

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/ronaldoussoren/pyobjc/blob/v8.5/docs/changelog.rst and  https://github.com/ronaldoussoren/pyobjc/releases
Requirements: 
- https://github.com/ronaldoussoren/pyobjc/blob/v8.5/pyobjc-core/setup.cfg
- https://github.com/ronaldoussoren/pyobjc/blob/v8.5/pyobjc-core/setup.py

Actions:
1. Add cbc.yaml
2. Add a patch, remove an outdated patch
3. Skip py<36
4. Update scripts
5. Update dependencies

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
NOTE: Remove `abs.yaml` before merging

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +37,6 @@ requirements:
   run:
     - python
     - libffi >={{ libffi }}
```
Comment (katietz) [inline recipe/meta.yaml]: is this really necessary?  should we simply use libffi as dependency here?

**Review by katietz — APPROVED** [review]
Otherwise look good to me

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: You are right, the plain `libffi` will be better. I've fixed it

**Review by katietz — APPROVED** [review]
thank you!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,19 +10,24 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:                # [osx and arm64]
-    - fix_ffi_fcfc.patch  # [osx and arm64]
+  patches:  # [osx]
```
Comment (marcoesters) [inline recipe/meta.yaml]: The `# [osx]` selector here seems redundant.

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Thanks. I've just removed the selector

**Pull request merged**